subsys/bootloader/mcuboot_image_header_parser/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources(mcuboot_image_header_parser.c)
+zephyr_library_include_directories($ENV{ZEPHYR_BASE}/../mcuboot/boot/bootutil/include/bootutil)
```

Comment:
Also open for a bettery way of hitting the MCUBoot directory from CMake.

Reply:
What if we update mcuboot/zephyr/CMakeLists.txt to create this library? Note that it has to be created outside the `if (CONFIG_BOOTLOADER_MCUBOOT)` check.

That way we might be able to get it upstreamed. Also this is not B0 - specific, but could be used for any application which interacts with mcuboot metadata.

Reply:
Can't everything outside b0 just use mcuboot's functions for this?